app/models/book.rb

```diff
@@ -49,7 +49,11 @@ def display_genres
   end
 
   def display_authors
-    authors.pluck(:name).join(", ")
+    if authors.count == 2
```

Comment:
In general, you want to prefer calling `.size` over `.count` or `.length` on activerecord objects.  http://web.archive.org/web/20100210204319/http://blog.hasmanythrough.com/2008/2/27/count-length-size for reference, but basically it figures out whether to count the ruby objects vs execute sql